index.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+require('dotenv').config();
+const mongoose = require ('mongoose'); 
+const app = require('./src/server.js');
+let PORT = process.env.PORT || 3000;
+let mongodb = process.env.MONGO_URL;
+
+
+
+
+
+
+
+mongoose.connect(mongodb, { useNewUrlParser: true, useUnifiedTopology: true })
```

Comment:
if you want you can move the `{ useNewUrlParser: true, useUnifiedTopology: true }`move  this into a variable like `const options = useNewUrlParser: true, useUnifiedTopology: true`